### PR TITLE
[HttpClient][Translation][Workflow] [Service]  Exclude tests from classmaps

### PR DESCRIPTION
**Pull request by danog**
| Q             | A
| ------------- | ---
| Branch?       | 6.1
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no

This excludes test directories from production classmaps.

### CODE REVIEW

File: src/Symfony/Contracts/Translation/composer.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
         "symfony/translation-implementation": ""
     },
     "autoload": {
-        "psr-4": { "Symfony\\Contracts\\Translation\\": "" }
+        "psr-4": { "Symfony\\Contracts\\Translation\\": "" },
+        "exclude-from-classmap": ["/Test/"]
```
Comment (nicolas-grekas) [inline src/Symfony/Contracts/Translation/composer.json]: for consistency with existing files, this should be on several lines

Comment (danog) [reply src/Symfony/Contracts/Translation/composer.json]: Done!

**Review by lyrixx — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @danog.

**Comment by GromNaN** [comment]
I think this is a breaking change for projects that use this classes and install dependencies with [authoritative class maps](https://getcomposer.org/doc/articles/autoloader-optimization.md#optimization-level-2-a-authoritative-class-maps), which is a good practice for performances.

**Comment by nicolas-grekas** [comment]
Tests + authoritative class maps looks like a mismatch to me. I don't think this will affect anyone. Let's see if anyone reports it later on :)

**Pull request closed without merging**